wdl/methods/phasing/PhysicalAndStatisticalPhasing.wdl

```diff
@@ -225,3 +232,39 @@ task FilterAndConcatVcfs {
         docker:"us.gcr.io/broad-dsp-lrma/lr-gcloud-samtools:0.1.2"
     }
 }
+
+
+# filter out singletons (i.e., keep MAC >= 2) and concatenate with deduplication
+task FilterPhasingInfo {
+
+    input {
+        File sv_vcf            # biallelic
```

Comment:
The `biallelic` comment is also vestigial.

If we want this to be a generic `UnphaseGenotypes` task that doesn't care if a VCF contains short/SV bi/multiallelic records, we should name the inputs in a generic fashion accordingly. If so, perhaps when we call the task we can alias it as `UnphaseSVGenotypes` to hint at the fact that we don't apply this operation to the short variants.